advocacy_docs/partner_docs/DBeaverGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for DBeaver'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | DBeaver       |
+| **Partner Product**   | DBeaver PRO: Lite, Enterprise, and Ultimate      |
+| **Web Site**      | https://dbeaver.com/     |
+| **Version**   | DBeaver Enterprise 22.0       |
+| **Product Description**      |   DBeaver is a SQL client software application and a database administration tool.    |
```

Comment:
DBeaver is a SQL client software application and a database administration tool for developers, database administrators, analysts and all people who work with Postgres.

Use this for description.